setup.py

```diff
@@ -13,13 +13,19 @@
     version="0.1.1",
     author="Jakob Vanhoefer, Marta R. A. Matos",
     author_email="marta.ra.matos@gmail.com",
-    description="A small package to convert ODEs specified in a yaml file to SBML/PEtab.",
+    description="A small package to convert ODEs specified in "
+                "a yaml file to SBML/PEtab.",
     url="https://github.com/martamatos/yaml2sbml",
     packages=setuptools.find_packages(),
     install_requires=["python-libsbml>=5.18.0",
                       "PyYAML>=5.1",
                       "pandas>=1.0.1",
                       "petab>=0.1.4"],
+    tests_require=["amici>=0.11.10",
```

Comment:
as you have this `test_requires`, you can also remove those pip installs in .github/workflows/install_deps.sh and move them here (pytest, pytest-cov, nbmake, ...) and there only install non-pip deps (for amici). Only the wheel package I would always install separately before all other stuff, because otherwise it gives nasty warnings.

Reply:
somehow this never worked for me, so I tried this out, but now they are in the `install_deps.sh` again ...

Reply:
hm, alternatively, you could define `extras_require={'tests': {'pytest>=...', ...}}` in setup.py and then do `pip install .[tests]`. but anything goes.